ide/Espressif/ESP-IDF/component-manager/LICENSE.txt

```diff
@@ -0,0 +1,354 @@
+
```

Comment:
Why is the GPLv2 in here? wolfSSH is GPLv3.

Reply:
This component install was copied & modified from wolfSSL. I missed the GPLv3 for wolfSSL vs GPLv2 wolfSSL